src/frontend/src/lib/components/settings/SettingsModalEnabledNetworks.svelte

```diff
@@ -0,0 +1,164 @@
+<script lang="ts">
+	import { Checkbox } from '@dfinity/gix-components';
+	import { isNullish } from '@dfinity/utils';
+	import {
+		SUPPORTED_MAINNET_NETWORKS,
+		SUPPORTED_NETWORKS,
+		SUPPORTED_TESTNET_NETWORKS
+	} from '$env/networks/networks.env';
+	import { ICP_NETWORK_ID } from '$env/networks/networks.icp.env';
+	import { setUserShowTestnets, updateUserNetworkSettings } from '$lib/api/backend.api';
+	import ManageNetworkToggle from '$lib/components/networks/ManageNetworkToggle.svelte';
+	import NetworkLogo from '$lib/components/networks/NetworkLogo.svelte';
+	import SettingsList from '$lib/components/settings/SettingsList.svelte';
+	import SettingsListItem from '$lib/components/settings/SettingsListItem.svelte';
+	import Button from '$lib/components/ui/Button.svelte';
+	import ButtonCloseModal from '$lib/components/ui/ButtonCloseModal.svelte';
+	import ButtonGroup from '$lib/components/ui/ButtonGroup.svelte';
+	import ContentWithToolbar from '$lib/components/ui/ContentWithToolbar.svelte';
+	import { authIdentity } from '$lib/derived/auth.derived';
+	import { isBusy } from '$lib/derived/busy.derived';
+	import { testnetsEnabled } from '$lib/derived/testnets.derived';
+	import { userNetworks } from '$lib/derived/user-networks.derived';
+	import { userProfileVersion } from '$lib/derived/user-profile.derived';
+	import { nullishSignOut } from '$lib/services/auth.services';
+	import { loadUserProfile } from '$lib/services/load-user-profile.services';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import type { Network } from '$lib/types/network';
+	import type { UserNetworks } from '$lib/types/user-networks';
+	import { emit } from '$lib/utils/events.utils';
+
+	let enabledNetworks = { ...$userNetworks };
+	const enabledNetworksInitial = { ...$userNetworks };
```

Comment:
to avoid possible different values, maybe is better...

```suggestion
	const enabledNetworksInitial = { ...enabledNetworks };
```